README.md

```diff
@@ -333,6 +334,13 @@ There is also an ongoing experimental project to make OCaml multiprocessor and m
 - [Volt](https://github.com/codinuum/volt) – A variant of Bolt OCaml logging tool.
 - [Logs](http://erratique.ch/software/logs) - Logs provides a logging infrastructure for OCaml.
 
+## Machine Learning
+
+- [Owl](https://ocaml.xyz/) - Scientifi library with neural networks, algoritmic differentiation and ONNX support.
+- [Deep Learning with OCaml](https://blog.janestreet.com/deep-learning-experiments-in-ocaml/).
+- [Transfer Learning with OCaml](https://blog.janestreet.com/of-pythons-and-camels/).
+- [Reinforcement Learning with OCaml](https://blog.janestreet.com/playing-atari-games-with-ocaml-and-deep-rl/).
```

Comment:
I think that bblog posts should be clearly marked as such, to avoid having people assume that the links point to separate libraries.

```
- [Owl](library link) (description)
  Example: [Object detection ..]
- PyTorch Bindings
  Example: blog post, blog post, blog post
```